src/js/components/stock-movement-wizard/SendMovementPage.jsx

```diff
@@ -382,9 +383,44 @@ class SendMovementPage extends Component {
     }
   }
 
-  render() {
-    const { previousPage } = this.props;
+  previousPage(values) {
+    if (values.dateShipped && values.shipmentType) {
```

Comment:
if you use validate like in other components you will have all validation in one place, you can use _.isEmpty to check if there are any errors